README.rst

```diff
@@ -42,6 +43,10 @@ We use compass to compiles scss files to css::
 
     compass compile
 
+After that collectstatic files::
+
+    python manage.py collectstatic
```

Comment:
I'd use ``python manage.py collectstatic -l``, which is  more suitable for local environment